pyproject.toml

```diff
@@ -22,6 +22,12 @@ requires-python = ">=3.7"
 dependencies = [
     "flask>=2.2",
     "cachelib",
+    "redis",
+    "pymongo",
+    "python3-memcached",
+    "Flask-SQLAlchemy",
+    # Testing
+    "nose",
```

Comment:
> Nose has been in maintenance mode for the past several years and will likely cease without a new person/team to take over maintainership. New projects should consider using [Nose2](https://github.com/nose-devs/nose2), [py.test](http://pytest.org/), or just plain unittest/unittest2.

I think it'd be better if someone just moved those tests to pytest straight away instead of adding this pretty much legacy test runner...

Reply:
Addressed in the latest commit